packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -128,101 +128,105 @@ const ChatReaction = ({ reaction }: ChatReactionProps) => {
-  shouldShowDate?: boolean
+  message: ChatMessageWithExtras
+  isPopup: boolean
   style?: StyleProp<ViewStyle>
-  onLongPress?: () => void
+  onLongPress?: (id: string) => void
+  itemsRef?: any
 }
 
-export const ChatMessageListItem = forwardRef<View, ChatMessageListItemProps>(
-  (props: ChatMessageListItemProps, refProp) => {
-    const {
-      message,
-      hasTail,
-      shouldShowReaction = true,
-      shouldShowDate = true,
-      style: styleProp,
-      onLongPress
-    } = props
-    const styles = useStyles()
-    const palette = useThemePalette()
+export const ChatMessageListItem = memo(function ChatMessageListItem(
+  props: ChatMessageListItemProps
+) {
+  const {
+    message,
+    isPopup = false,
+    style: styleProp,
+    onLongPress,
+    itemsRef
+  } = props
+  const styles = useStyles()
+  const palette = useThemePalette()
 
-    const userId = useSelector(getUserId)
-    const senderUserId = decodeHashId(message.sender_user_id)
-    const isAuthor = senderUserId === userId
+  const userId = useSelector(getUserId)
+  const senderUserId = decodeHashId(message.sender_user_id)
+  const isAuthor = senderUserId === userId
 
-    return (
-      <>
-        <View
-          style={[
-            isAuthor ? styles.rootIsAuthor : styles.rootOtherUser,
-            !hasTail && message.reactions && message.reactions.length > 0
-              ? styles.reactionMarginBottom
-              : null,
-            styleProp
-          ]}
-        >
-          <View>
-            <TouchableWithoutFeedback onPress={onLongPress}>
-              <View>
-                <View
-                  style={[styles.bubble, isAuthor && styles.isAuthor]}
-                  ref={refProp}
-                >
-                  <Text
-                    style={[styles.message, isAuthor && styles.messageIsAuthor]}
-                  >
-                    {message.message}
-                  </Text>
-                </View>
-                {message.reactions?.length > 0 ? (
-                  <>
-                    {shouldShowReaction ? (
-                      <View
-                        style={[
-                          styles.reactionContainer,
-                          isAuthor
-                            ? styles.reactionContainerIsAuthor
-                            : styles.reactionContainerOtherUser
-                        ]}
-                      >
-                        {message.reactions.map((reaction, index) => {
-                          return (
-                            <ChatReaction key={index} reaction={reaction} />
-                          )
-                        })}
-                      </View>
-                    ) : null}
-                  </>
-                ) : null}
-              </View>
-            </TouchableWithoutFeedback>
-          </View>
-          {hasTail ? (
-            <>
+  const handleLongPress = useCallback(() => {
+    return onLongPress ? onLongPress(message.message_id) : null
```

Comment:
nit: don't need `return` here. could just do `onLongPress?.(message.message_id)`

Reply:
Didn't know you could do that!